scipy/stats/_stats_py.py

```diff
@@ -4699,21 +4712,34 @@ def fisher_exact(table, alternative='two-sided'):
 
         See the Notes for more details.
 
+    method : ResamplingMethod, optional
+        Defines the method used to compute the p-value.
+        If `method` is an instance of `PermutationMethod`/`MonteCarloMethod`,
```

Comment:
Could you add the `rvs` must be unspecified

Reply:
And actually it might be a good idea for `chi2_contingency` too

Reply:
Yup, good idea.